server/app/lib/actions/fusor/deployment/openstack/cfme_launch.rb

```diff
@@ -97,7 +97,6 @@ def create_host(deployment)
                     "provision_method" => "image",
                     "build" => 1,
                     "is_owned_by" => "3-Users",
-                    "enabled" => 1,
```

Comment:
This was picked up by the new dupe hash index checking in rubocop. See http://git.io/vBpKY